lib/network.go

```diff
@@ -2797,29 +2770,12 @@ func (msg *MsgDeSoBlock) EncodeBlockVersion1(preSignature bool) ([]byte, error)
 	return data, nil
 }
 
```

Comment:
All of these deletions are amazing. There was so much complexity driven by the changing block header that is now no longer needed. I really love it.